Legacy/bonej/src/main/resources/plugins.config

```diff
@@ -33,7 +33,7 @@ Plugins>BoneJ>, "Fit Sphere", org.bonej.plugins.SphereFitter
 Plugins>BoneJ>, "Moments of Inertia", org.bonej.plugins.Moments
 Plugins>BoneJ>, "Slice Geometry", org.bonej.plugins.SliceGeometry
 
-Plugins>BoneJ>Stacks, "Delete Slice Range", org.bonej.plugins.DeleteSliceRange
-Plugins>BoneJ>Stacks, "Check Voxel Depth", org.bonej.plugins.VoxelDepthChecker
+Image>Stacks, "Delete Slice Range", org.bonej.plugins.DeleteSliceRange
```

Comment:
@alessandrofelder You are correct, this functionality is replicated in Fiji's Stack_Manipulation plugins, but not in vanilla ImageJ1 or 2.

https://github.com/fiji/Stack_Manipulation/tree/master/src/main/java

The code is essentially the same and is very small, and a general utility method. It would be worth upstreaming to core ImageJ. What do you think @imagejan @ctrueden ?

Until that happens we should leave Delete Slice Range in BoneJ2, so that it is present in vanilla ImageJ2 installations, at the cost of some redundancy for Fiji users (extra batteries??).

Reply:
Could someone file an issue somewhere so that we remember? Maybe this could go into `imagej-ops` or `imglib2-algorithm`?

Reply:
> Could someone file an issue somewhere

Happy to oblige:
https://github.com/imagej/imagej-ops/issues/582